automation/src/test/scala/org/broadinstitute/dsde/automation/config/Config.scala

```diff
@@ -0,0 +1,65 @@
+package org.broadinstitute.dsde.automation.config
```

Comment:
My intention here is like WebBrowserSpec: don't contain anything Firecloud-specific and don't depend on anything from the firecloud package.

I'm not sure I succeeded - maybe a few things like `appsDomain` and `smoketestuser` don't belong here

Reply:
the config seems to be pretty firecloud-specific in my opinion